epub33/rs/index.html

```diff
@@ -2303,19 +2301,19 @@ <h3>Recommendations</h3>
 					operation of an EPUB Publication, particularly on platforms where the sale of an EPUB Publication
 					and the method of reading it are connected. In these cases, it is recommended that the Reading
 					System or retailer be clear about the data being collected, how it is used, and allow for user
-					opt-outs where possible. Anonymization of data is strongly recommended for the privacy and the
+					opt-outs where possible. Anonymisation of data is strongly recommended for the privacy and the
```

Comment:
Fixing to the British spelling? 😉 
```suggestion
					opt-outs where possible. Anonymization of data is strongly recommended for the privacy and the
```